src/app/forms/services/dynamic-form.service.ts

```diff
@@ -122,7 +122,7 @@ export class DynamicFormService {
     validators.forEach(v => control.addAsyncValidators(this.asyncValidatorMap[v.name](v.args)));
   }
 
-  static updateValidity(form: CustomFormGroup | CustomFormArray, errors: GlobalError[]) {
+  static updateValidity(form: FormGroup | FormArray, errors: GlobalError[]) {
```

Comment:
why was this changed?

Reply:
You were on the call when I talked about htis change 😆, so because both custom types extend the base and now im using a base group this works and it allowed me to reuse it 👍

Reply:
As theyres nothing specific to either of those types used and the required cast was already there

Reply:
well shows I pay attention then 😆 yep looks good then 👍